### PR TITLE
Use event delegation to reduce number of event listeners

### PR DESCRIPTION
**Pull request by beru**

### CODE REVIEW

File: source/grapher.js [hunk 1]
```diff
@@ -811,24 +831,7 @@ grapher.Edge = class {
...
-                e.stopPropagation();
-            });
-        }
-        if (this.activate) {
-            this.hitTest.addEventListener('click', (e) => {
-                this.activate();
-                e.stopPropagation();
-            });
-        }
+        this.hitTest._var = this;
```
Comment (lutzroeder) [inline source/grapher.js:834]: Can this be accomplished without storing objects on HTML elements?

**Review by lutzroeder — CHANGES_REQUESTED** [review]
@beru can you provide some context how to reproduce this issue. Can you test the change in different browser and desktop environments.

File: source/grapher.js (hunk 1 above)
Comment (beru) [reply source/grapher.js]: I've updated the code. 2090f75554b5409170703ef28d5fe06016a1e59a

**Comment by lutzroeder** [comment]
@beru can you rebase to main and resolve the build issues?

**Pull request closed without merging**